src/app/page.tsx

```diff
@@ -1,111 +1,13 @@
 import Image from 'next/image'
+import MarketingMaterials from '@/components/MarketingMaterials'
+import englishImage from '/Users/krishmalik/Documents/inreach-website/src/app/images/english-images.png'; 
+import '/Users/krishmalik/Documents/inreach-website/src/app/globals.css';
+
 
 export default function Home() {
 	return (
-		<main className='flex min-h-screen flex-col items-center justify-between p-24'>
```

Comment:
Is this the boilerplate included in the Nextjs app, and is that why you are deleting it?